plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ComposeMachineProviderImpl.java

```diff
@@ -590,10 +593,28 @@ private void readContainerLogsInSeparateThread(String container,
                 } catch (ContainerNotFoundException e) {
                     isContainerRunning = false;
                 } catch (IOException e) {
-                    LOG.error("Failed to get logs from machine {} of workspace {} backed by container {}",
+                    LOG.warn("Failed to get logs from machine {} of workspace {} backed by container {}, because: {}.",
                               workspaceId,
                               machineId,
-                              container);
+                              container,
+                              e.getMessage(),
+                              e);
```

Comment:
Looks like LOG.warn is misused. This methods accepts 1 string and 1 throwable or multiple strings. Did you use it in such a way on purpose?